client/views/question.html

```diff
@@ -1,44 +1,101 @@
-          Start Game</br>with {{game.players.length}} players
-        </div>
-      </div>
-    </div>
-    <!-- Game Over Stuff -->
-    <div id="game-end-info">
-      <div id="people-left-game-top" ng-show="game.state ==='game dissolved' && game.gameWinner === -1">
-        <div class="game-end-headline">What the!? The game ended because too many people left! </div>
-        Our sincere apologies.
-      </div>
-      <div ng-show="game.state === 'game ended' && game.gameWinner === game.playerIndex">
-        <div class="game-end-headline">Congratulations! You won the game, {{game.players[game.playerIndex].username | upperFirstLetter}}!</div>We hope you feel special. If you'd like to share that special feeling with the world, consider donating to charity. Or just horde it all for youself.
-      </div>
-      <div ng-show="game.state === 'game ended' && game.gameWinner !== game.playerIndex">
-        <div class="game-end-headline">Aw, shucks! {{game.players[game.gameWinner].username | upperFirstLetter }} won the game!</div>
-        That makes you a loser, unfortunately.
-      </div>
+<!-- searching -->
+<div
+  class="waiting"
+  ng-show="game.state === 'awaiting players'"
+>
+  <div class="waiting-text">
+    <p>Searching for Players</p>
+    <div>
+      <img src="/img/loaders/spinner.svg" width="50"/>
     </div>
+    <p class="text-center">{{game.players.length}}&nbsp;/&nbsp;12 Players Found</p>
+  </div>
+  <div class="waiting-btns">
+    <button
+      class="waves-effect waves-light btn start-game-btn"
+      ng-click="game.startGame()"
+    >
+      Start Game
+    </button>
+    <button
+      class="waves-effect waves-light btn invite-btn"
+      ng-click="showModal('.invite-modal')"
+    >
+      Invite Players
+    </button>
+  </div>
+</div>
+
+<!-- question -->
+<span
+  class="question"
+  ng-bind-html-unsafe="game.curQuestion.text"
+  ng-hide="game.state === 'awaiting players' || game.state === 'game ended' || game.state === 'game dissolved'"
+>
+</span>
+
+<!-- When game is dissolved -->
+<div
+  class="question"
+  ng-show="game.state === 'game dissolved' && game.gameWinner === -1"
+>
+  What the!? The game ended because too many people left!
+  Our sincere apologies.
+</div>
+
+<!-- When game ends -->
+<div
+  class="waiting"
+  ng-show="game.state === 'game ended'">
+  <div
+    class="winner-text"
+    ng-show="game.gameWinner === game.playerIndex"
+  >
+    <p style="font-size: 1.7em">Congratulations! You won the game,
+      <span style="color: #e5b05b">
+        {{game.players[game.playerIndex].username | upperFirstLetter}}!
+      </span>
+    </p>
+    <p>
+      We hope you feel special. If you'd like to share that special feeling with the world, consider donating to charity. Or just horde it all for youself.
+    <p>
+  </div>
+  <div
+    class="loser-text"
+    ng-show="game.gameWinner !== game.playerIndex"
+    style="font-size: 1.7em"
+  >
+    Aw, shucks!
+    <span style="color: #e5b05b">
```

Comment:
The team agreed to avoid inline styles. Kindly extract the inline styles into classes in a css file.

Reply:
Thanks for the feedback. Will make corrections and ensure I don't make such mistakes in future.

Reply:
Fixed!